recipes/wasmer/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "wasmer" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wasmerio/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: ca5ea30bfd0700b5f89dace19516991dbc9ef38cf451f5bf11cab283e5f12777
+
+build:
+  # TODO: determine if we can use a newer OSX, or somehow push upstream on the missing pieces
+  skip: true  # [osx]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - rust >=1.47  # [not osx]
+    - rust ==1.47  # [osx]
+    - llvmdev >=10
+  host:
+    - libffi
+    - openssl
+    - zlib
+  run: []
+
+
+# TODO: Also Included in the 1.0.1 release is an experimental, minified version
+#       of wasmer called wasmer-headless that can only run precompiled Wasm modules.
+# outputs:
+
+# TODO: ideally have some "canonical" WASM to run...
+{% set test_wasm = "tests/wasi-wast/wasi/snapshot1/hello.wasm" %}
+{% set test_text = "Hello, world!" %}
+
+test:
+  source_files:
+    - {{ name }}-{{ version }}-cargo-dependencies.json
+    - {{ test_wasm }}
+  files:
+    # TODO: we should probably generalize and package this as e.g.
+    #       pytest-(conda-)cargo-liceneses, once we get cargo-licenses built
+    - test_licenses.py
+  requires:
+    - python >=3.6
+    - pytest
+    - ruamel_yaml
+  commands:
+    - wasmer --version
+    - wasmer --help
+    # TODO: remove jinja templates
+    - wasmer validate {{ test_wasm }}
+    - wasmer run {{ test_wasm }}
+    - wasmer run {{ test_wasm }} | grep '{{ test_text }}'
+    - python test_licenses.py --ignore wasmer
+    # TODO: enable C API, multiple outputs, use python testing infra, etc.
+    # - mkdir .wsmr
+    # {% for backend in ["singlepass", "cranelift", "llvm"] %}
+    # - wasmer run --{{ backend }} {{ test_wasm }}
+    # - wasmer run --{{ backend }} {{ test_wasm }} | grep '{{ test_text }}'
+    # - WASMER_DIR=$PREFIX wasmer create-exe --{{ backend }} {{ test_wasm }} -o ./hello-{{ backend }}
+    # - ls -lathr hello-{{ backend }}
+    # - ./hello-{{ backend }}
+    # - ./hello-{{ backend }} | grep '{{ test_text }}'
+    # {% endfor %}
+
+about:
+  home: https://wasmer.io
+  license: MIT
+  license_family: MIT
+  license_file:
+    - LICENSE
+    # TODO: this is... some of the third-party licenses... need the whole rust license toolchain
+    - ATTRIBUTIONS.md
+  summary: Universal WebAssembly runtime
+  description: |
+    Wasmer is a standalone JIT WebAssembly runtime, aiming to be fully
+    compatible with WASI and Emscripten.
+  dev_url: https://github.com/wasmerio/wasmer
+  doc_url: https://docs.wasmer.io
+  doc_source_url: https://github.com/wasmerio/docs.wasmer.io
+
+extra:
+  recipe-maintainers:
+    - bollwyvl
+    - wolfv
```

Comment:
@wolfv can I get a confirmation here?

Also, if you are (or anybody is!) dissatisfied with my pace of development or particular choices, please feel free to either:
- brutally review
- PR (or push at, i guess if you're CF core :laughing:) at this
- Start a new PR, and we can trade back and forth

Reply:
sure, confirming! I am really happy with everything you do ;) keep on pushing!